packages/yew/src/virtual_dom/vtag.rs

```diff
@@ -430,8 +431,28 @@ impl VTag {
             .insert(key, value.into_prop_value());
     }
 
-    /// Set event listeners on the [VTag]'s  [Element]
+    /// Add pending event listener on the [VTag]'s  [Element]
+    pub fn add_pending_listener(&mut self, listener: Rc<dyn Listener>) -> bool {
+        if let Listeners::Pending(listeners) = &mut self.listeners {
+            let mut listeners = mem::take(listeners).into_vec();
+            listeners.push(Some(listener));
+
+            self.set_listeners(listeners.into_boxed_slice());
+            true
+        } else {
+            false
+        }
+    }
+
+    #[deprecated(
+        note = "The `set_listener` method is deprecated and will be removed in the future. Use the `set_listeners` method instead."
+    )]
```

Comment:
Let's just remove this function instead of deprecating this. Breaking changes are fine

Reply:
Yeah what is "pending event listener"?

Reply:
@hamza1311 Ok, I removed this function.